cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -211,6 +215,8 @@ static ImmutableArguments.Builder defaultsBuilder(final String profileString) {
 
     boolean anonymize();
 
+    boolean usePrivateResources();
```

Comment:
This is a little awkward. Now users need to remember to have this flag set when they use the private image, and its not clear that this is very specific to cuppa. Could we think of a way to do it dynamically based on whether the private resource exists? Or have an empty one in the public image to remove the optionality?